src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -72,9 +77,13 @@ private ResponseEntity postEventInternal(final String eventTypeName,
             final JSONArray eventsAsJsonObjects = new JSONArray(eventsAsString);
 
             final int eventCount = eventsAsJsonObjects.length();
-            eventTypeMetrics.reportSizing(eventCount, eventsAsString.length());
```

Comment:
There were bug, because String.length() doesn't correspond amount of bytes in string.